packages/destination-actions/src/destinations/sendgrid/updateUserProfile/index.ts

```diff
@@ -0,0 +1,251 @@
+      description: `The user's country`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.country' },
+          then: { '@path': '$.traits.address.country' },
+          else: { '@path': '$.properties.address.country' }
+        }
+      }
+    },
+    postal_code: {
+      label: 'Postal Code',
+      description: `The user's postal code`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.postal_code' },
+          then: { '@path': '$.traits.address.postal_code' },
+          else: { '@path': '$.properties.address.postal_code' }
+        }
+      }
+    },
+    city: {
+      label: 'City',
+      description: `The user's city`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.city' },
+          then: { '@path': '$.traits.address.city' },
+          else: { '@path': '$.properties.address.city' }
+        }
+      }
+    },
+    state: {
+      label: 'State',
+      description: `The user's state`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.state' },
+          then: { '@path': '$.traits.address.state' },
+          else: { '@path': '$.properties.address.state' }
+        }
+      }
+    },
+    address_line_1: {
+      label: 'Address Line 1',
+      description: `The user's address line 1`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.line1' },
+          then: { '@path': '$.traits.address.line1' },
+          else: { '@path': '$.properties.address.line1' }
+        }
+      }
+    },
+    address_line_2: {
+      label: 'Address Line 2',
+      description: `The user's address line 2`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.address.line2' },
+          then: { '@path': '$.traits.address.line2' },
+          else: { '@path': '$.properties.address.line2' }
+        }
+      }
+    },
+    phone_number: {
+      label: 'Phone Number',
+      description: `The user's phone number`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.phone_number' },
+          then: { '@path': '$.traits.phone_number' },
+          else: { '@path': '$.properties.phone_number' }
+        }
+      }
+    },
+    whatsapp: {
+      label: 'whatsapp',
+      description: `The user's whatsapp`,
+      type: 'string',
+      allowNull: true,
+      default: {
+        '@if': {
+          exists: { '@path': '$.traits.whatsapp' },
+          then: { '@path': '$.traits.whatsapp' },
+          else: { '@path': '$.properties.whatsapp' }
+        }
+      }
+    },
+    line: {
+      label: 'Line Id',
```

Comment:
I'm pretty sure LINE is capitalized.

```suggestion
      label: 'LINE ID',
```